From ad72be8d24fd9a431422865ab5cc34669c547216 Mon Sep 17 00:00:00 2001
From: Yingda WU <yingda.wu@hpmicro.com>
Date: Wed, 14 Aug 2024 10:30:18 +0800
Subject: [PATCH 2/2] patch: conflict "display_pixel_format" between sdk and
 zephyr

- change sdk "display_pixel_format" to "hpm_display_pixel_format"

Signed-off-by: Yingda WU <yingda.wu@hpmicro.com>
---
 hpm_sdk/drivers/inc/hpm_display_common.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hpm_sdk/drivers/inc/hpm_display_common.h b/hpm_sdk/drivers/inc/hpm_display_common.h
index 0e576e25..2503bf11 100644
--- a/hpm_sdk/drivers/inc/hpm_display_common.h
+++ b/hpm_sdk/drivers/inc/hpm_display_common.h
@@ -42,7 +42,7 @@ typedef enum display_alphablend_mode {
 /**
  * @brief display pixel format
  */
-typedef enum display_pixel_format {
+typedef enum hpm_display_pixel_format {
     display_pixel_format_argb8888,
     display_pixel_format_rgb565,
     display_pixel_format_rgb555,
-- 
2.25.1

